tms/clickup/views.py

```diff
@@ -25,6 +32,10 @@ def get_login_page(request):
     return render(request, "login.html")
 
 
+def get_create_project_page(request):
+    return render(request, "project.html")
+
+
 class SingUpView(CreateAPIView):
     serializer_class = RegistrationSerializer
     permission_classes = [AllowAny]
```

Comment:
In sign up[ api view why Permission class ? If I will not write permission class then  ?